Formula/trojan-go.rb

```diff
@@ -0,0 +1,70 @@
+class TrojanGo < Formula
+  desc "Trojan proxy in Go"
+  homepage "https://p4gefau1t.github.io/trojan-go/"
+  url "https://github.com/p4gefau1t/trojan-go/archive/v0.10.0.tar.gz"
+  sha256 "423fa05d81da0105a01682a77bbbd95fef09644c3a9dbbddd641436e31879429"
+  license "GPL-3.0-only"
+  head "https://github.com/p4gefau1t/trojan-go.git"
+
+  depends_on "go" => :build
+
+  resource "geoip" do
+    url "https://raw.githubusercontent.com/v2fly/geoip/release/geoip.dat"
+  end
+
+  resource "geosite" do
+    url "https://raw.githubusercontent.com/v2fly/domain-list-community/release/dlc.dat"
+  end
```

Comment:
This is fine if you're leaving it like this for testing, but these will need a versioned URL and checksums before merging.

Reply:
I suggest something like:
```
  resource "geoip" do
    url "https://github.com/v2fly/geoip/releases/download/202105070029/geoip.dat"
    sha256 "f3aeb80c21fd403c6030988477d0cac7574322b786733d0ecbc24ff4ab55ce93"
  end

  resource "geosite" do
    url "https://github.com/v2fly/domain-list-community/releases/download/20210507022421/dlc.dat"
    sha256 "c08b57f45ad248e3812ff6c453c788c85916cd443d4c11e26ccd1bb6919c928d"
  end
```
Those are the latest as of this writing.

Reply:
Thank you @gromgit . I'll change these code.
In the long run, these does not seem to be the best solution.

Reply:
Then perhaps you should file an issue with upstream. As far as I can tell, these two files are only pulled at install time, with no script or other instructions for updating them over time.

Reply:
Thanks. I am not a developer of upstream code, this may only be done by them. 🤣

Reply:
After trying I still don't know how to separate these two resources into formula and let the program read them. 😞

Reply:
It's okay to leave them as resources for now.